src/components/presentations/Article/ArticlePage.jsx

```diff
@@ -42,7 +48,23 @@ class ArticlePage extends Component {
                     </div>
                   </Grid.Column>
                   <Grid.Column>
-                    <Comments comments={article.Comments} />
+                    <h3>Comments</h3>
+                    {article && article.author && (
```

Comment:
To check that an object is empty we can use `isEmpty` lodash - `isEmpty(obj)`

Alternatively,

Create a util function that checks that the object does not have any keys to ensure that it is empty